persistence/src/main/scala/hmda/persistence/institutions/InstitutionPersistence.scala

```diff
@@ -56,6 +56,7 @@ class InstitutionPersistence extends HmdaPersistentActor {
         persist(InstitutionCreated(i)) { e =>
           log.debug(s"Persisted: $i")
           updateState(e)
+          sender() ! e.i
```

Comment:
Now that this is sending a response when an institution is added `DemoData` should probably be updated to accept or at least acknowledge this response. I'll make another ticket for it though as the only effect I can see is that the log displays 4 dead letters when demo is set to true.